webgrid/flask.py

```diff
@@ -1,6 +1,14 @@
 from __future__ import absolute_import
 
-from flask import request, session, flash, Blueprint, url_for, send_file
+from flask import (
```

Comment:
nit: we probably should have just done `import flask` to begin with. Oh well. No real need to change it right now.

Reply:
I'll go ahead and do this, as it is a major improvement in readability.